src/wagmiConfig/index.ts

```diff
@@ -184,6 +185,7 @@ export const wagmiConfig = createConfig({
   connectors: process.env.NEXT_PUBLIC_MOCK_CONNECTOR
     ? [
         mock({
+          // WARNING: environment variable probably doesn't exist
```

Comment:
Yeah, well, it won't exist on Vercel, we only use it locally & on GitHub for running our E2E tests.